desc/equilibrium.py

```diff
@@ -265,6 +270,108 @@ def change_resolution(
         if N_grid is not None and N_grid != self.N_grid:
             self._N_grid = N_grid
 
+    @classmethod
+    def from_near_axis(
+        cls, na_eq, r=0.1, L=None, M=8, N=None, ntheta=None, spectral_indexing="ansi"
+    ):
+        """Initialize an Equilibrium from a near-axis solution.
+
+        Parameters
+        ----------
+        na_eq : Qsc or Qic
+            Near-axis solution generated by pyQSC or pyQIC.
+        r : float
+            Radius of the desired boundary surface (in meters).
+        L : int (optional)
+            Radial resolution. Default 2*M for `spectral_indexing`==fringe, else M
+        M : int (optional)
+            Poloidal resolution. Default is 8
+        N : int (optional)
+            Toroidal resolution. Default is the max resolution provided by na_eq.nphi
```

Comment:
could this potentially be an issue if nphi is super large?

Reply:
Yes. But I want the option to be able to capture the full toroidal resolution from the near-axis solution if necessary. We could change the default to `N=8` or something?

Reply:
could maybe make it default to M==N, but have N=np.inf mean use all of them?